rust/bindings_list

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: GPL-2.0
+
+--opaque-type xregs_state
+--opaque-type desc_struct
+--opaque-type arch_lbr_state
+--opaque-type local_apic
```

Comment:
Perhaps we could call the file `rust/bindgen_parameters`.

Or perhaps `rust/bindgen_opaque_types` and then we remove `--opaque-type` here and keep the `addprefix` in the `Makefile` etc. Not sure.

Reply:
I am not certain, but there is a possibility that we might want to use blocklist in the future.